server/notification-providers/dingding.js

```diff
@@ -11,17 +11,22 @@ class DingDing extends NotificationProvider {
      */
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
         const okMsg = "Sent Successfully.";
-
+        const mentionAll = notification.mentioning === "everyone";
+        const mobileList = notification.mentioning === "specify-mobiles" ? notification.mobileList.split(",") : [];
+        const userList = notification.mentioning === "specify-users" ? notification.userList.split(",") : [];
+        const mentionStr = [ ...mobileList || [], ...userList || [] ].map(item => `@${item}`).join(" ");
```

Comment:
please handle users
- using spaces => trimming the input
- using `@` => trimming the input unless this is allowed
- (optional) drop duplicates from the list

Reply:
Thx for review, i'll change it.